encrypted-media/content/drmconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "com.widevine.alpha": {
+        "drmtoday": true,
+        "serverURL": "https://lic.staging.drmtoday.com/license-proxy-widevine/cenc/",
+        "httpRequestHeaders": {
+            "dt-custom-data": "eyJ1c2VySWQiOiIxMjM0NSIsInNlc3Npb25JZCI6ImV3b2dJQ0p3Y205bWFXeGxJaUE2SUhzS0lDQWdJQ0p3ZFhKamFHRnpaU0lnT2lCN0lIMEtJQ0I5TEFvZ0lDSnZkWFJ3ZFhSUWNtOTBaV04wYVc5dUlpQTZJSHNLSUNBZ0lDSmthV2RwZEdGc0lpQTZJR1poYkhObExBb2dJQ0FnSW1GdVlXeHZaM1ZsSWlBNklHWmhiSE5sTEFvZ0lDQWdJbVZ1Wm05eVkyVWlJRG9nWm1Gc2MyVUtJQ0I5TEFvZ0lDSnpkRzl5WlV4cFkyVnVjMlVpSURvZ1ptRnNjMlVLZlFvSyIsIm1lcmNoYW50IjoiY2FibGVsYWJzIn0K"
```

Comment:
@mwatson2 Keep in mind, the tokens have to be formatted exactly like this in the HTTP headers.  This is required by the DRMToday system.  The internal base64 token (stored in the "sessionID" field) is actual the "rights token" that the authentication server would normally compute on its own and return to DRMToday.  We may need build custom rights tokens to support things like persistent sessions.  But again, this can be done just by modifying this field with no modifications to the license server or authentication server.  I work with the team after this initial work to understand how we can make further use of this field.